### DISCUSSION TITLE
Collaborator issue

### DISCUSSION THREAD

**Discussion by pankaj86999646** [community/community#77287] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when i share my repo (not sharing the organization) to any other user and make him or her as collaborator then after accepting the invitation, collaborator is not able to see repo in repository section where all repository will display

**Comment by MantasTech** [comment]
Hello pankaj,

1. Direct Link: Send them the direct link to the repo.
2. Refresh: Sometimes it just needs a refresh or re-login.
3. Check 'Repositories' Tab: They should look under the 'Repositories' tab in their profile, not just the dashboard.
4. Repo Visibility: Make sure the repo isn't private to the organization.

If it still doesn't show up, there might be a permission issue. Double-check the repo settings. 👍

**Comment by Varsha-JJ** [comment]
Check the repository settings to ensure that the collaborator has the necessary permissions.
